src/manifest/net/manifest-net-add-outbound.action.ts

```diff
@@ -16,16 +16,24 @@ function parseUrls(urls: string[]): URL[] {
 
   urls.forEach((url) => {
     try {
-      parsed.push(new URL(url));
+      const parsedUrl = new URL(url);
+      if (!parsedUrl.protocol) {
```

Comment:
Doesn't appear to be working on my end. I deleted package-lock.json, node_modules, dist and ran `npm i && npm run build && cd dist && npm i && node main.js manifest create -w -i ubuntu:latest google.com` 

```bash
➜  dist git:(bugfix/JST-415) ✗ node main.js manifest create -w -i ubuntu:latest google.com
Created manifest in ./manifest.json file
```


```json
{
  "version": "0.1.0",
  "createdAt": "2023-09-14T09:21:30.074+02:00",
  "expiresAt": "2023-12-13T09:21:30.074+01:00",
  "metadata": {
    "name": "@golem-sdk/cli",
    "description": "CLI for Golem SDK",
    "version": "1.0.0"
  },
  "payload": [
    {
      "platform": {
        "os": "linux",
        "arch": "x86_64"
      },
      "hash": "ubuntu:latest",
      "urls": [
        "google.com"
      ]
    }
  ]
}
```